packages/kubernetes-1.21/kubelet.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
```

Comment:
Do kubelet logs look different?  Is there an additional log file?  https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md#structured-logging-in-kubelet

Reply:
The logs do not look different from k8s-1.20. Not sure if this is something we have to enable.